metrics/grafana/tidb.json

```diff
@@ -8249,22 +8249,22 @@
           "steppedLine": false,
           "targets": [
             {
-              "expr": "histogram_quantile(0.999, sum(rate(tidb_tikvclient_ts_future_wait_seconds_bucket{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", type!~\"tso|tso_async_wait\"}[1m])) by (le, type))",
```

Comment:
`tidb_tikvclient_ts_future_wait_seconds_bucket` wraps the wait function of PD's ts future, and the metrics miss observing some tso requests, I'm planning to remove it after this PR.